### ISSUE TITLE
feat: breadcrumbs - use link for all items

### ISSUE THREAD

**Issue by dargmuesli** [harlan-zw/nuxt-seo-ui#3]
### Clear and concise description of the problem

The last breadcrumb item is a span with `role="link"` which does not satisfy [an accessibility recommendation](https://html-validate.org/rules/prefer-native-element.html).

### Suggested solution

I'd like to propose to use links for all items.

**Comment by harlan-zw** [comment]
Hey, thank you for flagging this. I needed to do some more testing.

I've changed the tag to `a`, it should be accurate in 0.4.2. The accessibility is based off of https://spectrum.adobe.com/page/breadcrumbs/

Let me know if there's still issues though
